tests/integration/targets/cloudhsm/files/signingCA.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
private keys should never be committed, doesn't matter if it's just a testing key ...  
It must be created on the fly with https://docs.ansible.com/ansible/latest/collections/community/crypto/openssl_privatekey_module.html

Reply:
Fixed!